moto/managedblockchain/utils.py

```diff
@@ -23,7 +28,73 @@ def get_network_id():
     )
 
 
+def memberid_from_managedblockchain_url(full_url):
+    id_search = re.search("\/m-[A-Z0-9]{26}", full_url, re.IGNORECASE)
+    return_id = None
+    if id_search:
+        return_id = id_search.group(0).replace("/", "")
+    return return_id
+
+
 def get_member_id():
     return "m-" + "".join(
         random.choice(string.ascii_uppercase + string.digits) for _ in range(26)
     )
+
+
+def proposalid_from_managedblockchain_url(full_url):
+    id_search = re.search("\/p-[A-Z0-9]{26}", full_url, re.IGNORECASE)
+    return_id = None
+    if id_search:
+        return_id = id_search.group(0).replace("/", "")
+    return return_id
+
+
+def get_proposal_id():
+    return "p-" + "".join(
+        random.choice(string.ascii_uppercase + string.digits) for _ in range(26)
+    )
+
+
+def invitationid_from_managedblockchain_url(full_url):
+    id_search = re.search("\/in-[A-Z0-9]{26}", full_url, re.IGNORECASE)
+    return_id = None
+    if id_search:
+        return_id = id_search.group(0).replace("/", "")
+    return return_id
+
+
+def get_invitation_id():
+    return "in-" + "".join(
+        random.choice(string.ascii_uppercase + string.digits) for _ in range(26)
+    )
+
+
+def member_name_exist_in_network(members, networkid, membername):
+    membernamexists = False
+    for member_id in members:
+        if members.get(member_id).network_id == networkid:
+            if members.get(member_id).name == membername:
+                membernamexists = True
+                break
+    return membernamexists
+
+
+def number_of_members_in_network(members, networkid):
+    return len(
+        [membid for membid in members if members.get(membid).network_id == networkid]
+    )
+
+
+def admin_password_ok(password):
```

Comment:
Doesn't look like the return value means anything - lets just return a boolean here.
I guess the comment was more a reminder to yourself?

Reply:
Yeah, I thought it would be more complex with different exceptions for the cases - turns out it is not.  Will update and send an update later on.  Have a couple other additions once I skip a test during server mode tests (freeze_time and expired proposals).

Reply:
Updated